plugins/verysmart.py

```diff
@@ -0,0 +1,12 @@
+import codecs
+import os
+import random
+
+from cloudbot import hook
+
+@hook.command('smart', 'verysmart', 'vs', 'iamverysmart', 'iavs', autohelp=False)
+def smart(bot):
+    with codecs.open(os.path.join(bot.data_dir, "verysmart.txt"), encoding="utf-8") as fp:  
```

Comment:
Load the data once in an `on_load` hook and just access that from the command., Opening the file every time someone runs a command is not a good idea.